_includes/nav.html

```diff
@@ -8,7 +8,7 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <a class="navbar-brand" href="{{ site.root }}/">{{ site.name }}</a>
+      <a href="https://cyboxproject.github.io" class="pull-left"><img src="/images/logo.png" style="max-height: 100px; max-width: 120px; margin-top: 3px;"></a>
```

Comment:
unfortunately `site.root`doesn't resolve to the base URL (just an empty string), so it's basically just a no-op